plugins/inputs/socketstat/README.md

```diff
@@ -0,0 +1,66 @@
+# SocketStat plugin
+
+The socketstat plugin gathers indicators from established connections, using iproute2's `ss` command.
+
+The `ss` command does not require specific privileges.
+
+The output format will have a very high cardinality. It should either be stored by an engine which doesn't suffer from it, of with a short retention policy.
+
+### Configuration
+
+```toml
+[[inputs.socketstat]]
+  ## ss can display information about tcp, udp, raw, unix, packet, dccp and sctp sockets
+  ## Specify here the types you want to gather
+  socket_types = [ "tcp", "udp" ]
+  ## The default timeout of 1s for ss execution can be overridden here:
+  # timeout = "1s"
+```
+
+### Measurements & Fields:
+
+- socketstat
+    - state (string) (tcp, dccp and sctp)
+    - If ss provides it (it depends on the protocol and ss version):
+        - bytes_acked (integer, bytes)
+        - bytes_received (integer, bytes)
+        - segs_out (integer, count)
+        - segs_in (integer, count)
+        - data_segs_out (integer, count)
+        - data_segs_in (integer, count)
+
+### Tags:
+
+- All measurements have the following tags:
+    - proto
+    - state (tcp, dccp and sctp)
```

Comment:
`state` is already a field and in the light of exploding cardinality we might want to remove the tag!?

Reply:
This was  a typo, there is no state tag (only a field). Will fix.

Reply:
now fixed.